package.json

```diff
@@ -1,5 +1,21 @@
 {
-  "name": "ux-onboarding-project",
+  "name": "nathan-phillips-ux-iop",
   "version": "1.0.0",
-
+  "main": "index.html",
+  "dependencies": {
+    "server": "^1.0.18"
```

Comment:
It didn't work for me either until I changed this to http-server, and even then it wanted the stylesheet to be in a different place. I deleted the whole project, cloned it and tried to fire it up and got the same result